install_config/configuring_authentication.adoc

```diff
@@ -1391,3 +1391,55 @@ a secret with 32 or 64 bytes.
 <3> Encrypting secret, used to encrypt sessions. Must be 16, 24, or 32
 characters long, to select AES-128, AES-192, or AES-256.
 ====
+
+[[configuring-authentication-security]]
+== Security
+
+Administrators can prevent clients from accessing the API by their `User-Agent`
+header in the new `*userAgentMatching*` configuration setting.
+
+For example, you can deny clients using particular libraries or binaries. This
+example denies the Kubernetes 1.2 client binary, OpenShift Origin 1.1.3
+binary, and the POST and PUT *httpVerbs*:
+
+====
+----
+policyConfig:
+  userAgentMatchingConfig:
+    defaultRejectionMessage: "Your client is too old.  Go to https://example.org to update it."
+    deniedClients:
+    - regex: '\w+/v(?:(?:1\.1\.1)|(?:1\.0\.1)) \(.+/.+\) openshift/\w{7}'
+    - regex: '\w+/v(?:1\.1\.3) \(.+/.+\) openshift/\w{7}'
+    - regex: '\w+/v1\.2\.0 \(.+/.+\) kubernetes/\w{7}'
+      httpVerbs:
+      - POST
+      - PUT
```

Comment:
`httpVerbs` takes effect only upon its corresponding `regex`. From your example, the config will deny POST/PUT requests from Kubernetes 1.2, and deny all requests (not just POST/PUT) from Origin 1.1.3.
> +example denies the Kubernetes 1.2 client binary, OpenShift Origin 1.1.3
> +binary, and the POST and PUT *httpVerbs*:

The wording here seems to indicate: for Origin 1.1.3, it just denies POST/PUT. My suggestion is: specify httpVerbs after Line 1412 as well so that the wording says precisely.